scripts/fp_assessments_sum.R

```diff
@@ -0,0 +1,52 @@
+# this script gets a total count of fish passage field assessments done
+
+# xref my_crossings pscis, pull records from bcdata using record ID
+
+get_this <- bcdata::bcdc_tidy_resources('7ecfafa6-5e18-48cd-8d9b-eae5b5ea2881') %>%
```

Comment:
this mod doesn't actually do anything as written.... the purpose of this move is to get the "package_id" using the name of the layer from the BC data catalogue url (string after the last slash in the url). you can see the catalogue with `bcdc_browse()`